### DISCUSSION TITLE
Dashboard With Settings Using Mongodb Per Server

### DISCUSSION THREAD

**Discussion by VincentRPS** [DevBase-Dev/twigbot#8] [Ideas]
Will A Dashboard Ever Be Added And If So How Would It Be Implemented, I Think To Save Space For Other Files There Should Not Be A Config File As Once There's Alot Of Servers That File Would Be Massive It Would Be Easier To Implement MongoDB As The Settings But As A Dashboard Is Definitely Not Implementable Currently As We Don't Have Enough Features For That Once We Do Thought This Could Be In Talks As Having To Do h!config Over And Over Again Would Be Annoying, Especially In An Economy System Store's And Items Should Be Makeable And Editable Via The Dashboard

**Comment by techpluscraft** [comment]
Do you know how to do that
